modules/@apostrophecms/event-page/views/filters.html

```diff
@@ -0,0 +1,40 @@
+{%- macro here(url, changes) -%}
+  {{ url | build({
+    year: data.query.year
+  }, changes) }}
+{%- endmacro -%}
+
+{% fragment render(data) %}
+  <h3>Year</h3>
+  <ul>
+    {% for year in data.filters.year %}
+      <li>
+        <a
+          style="{{ 'font-style: italic' if data.query.year == year.value }}"
+          href="{{ here(data.url, { year: year.value }) }}">{{ year.label }}</a>
+      </li>
+    {% endfor %}
+  </ul>
+
+  <h3>Month</h3>
+  <ul>
+    {% for month in data.filters.month %}
+      <li>
+        <a
+          style="{{ 'font-style: italic' if data.query.month == month.value }}"
+          href="{{ here(data.url, { month: month.value }) }}">{{ month.label }}</a>
+      </li>
+    {% endfor %}
+  </ul>
+
+  <h3>Day</h3>
+  <ul>
+    {% for day in data.filters.day %}
```

Comment:
We could use select fields for the sample. That might be better, but it'd require client-side JS.